### DISCUSSION TITLE
ECHIPA MOGOSA

### DISCUSSION THREAD

**Discussion by sebiboga** [peviitor-ro/community#115] [TEAMs]
Dan Marius
role: Scraper Developer with Python
discord: danmarius7

**Comment by sebiboga** [comment]
Rafay Beatrice
role: Manual Tester
discord:RafayBeatrice

**Comment by AdrianaCionca** [comment]
Mihai Rata
role: Team Lead & Automation Tester with Python
discord: Mihai
